index.bs

```diff
@@ -498,7 +498,69 @@ Add the step:
 
 # Fenced Frame monkeypatches # {#fenced-frame-monkeypatches}
 
-Issue: Modify [=send a beacon=] when the [=fetch=] integration is specified.
+Modify {{Fence/reportEvent()}} as follows:
+
+Replace the step
+
+> Run <a spec=fenced-frame>report an event</a> using <var ignore>instance</var>'s
+> [=fenced frame config instance/fenced frame reporter=] with |destination|,
+> |event|'s {{FenceEvent/eventType}}, and |event|'s {{FenceEvent/eventData}}.
+
+with
+
+Run <a spec=fenced-frame>report an event</a> using <var ignore>instance</var>'s
+[=fenced frame config instance/fenced frame reporter=] with |destination|,
+|event|'s {{FenceEvent/eventType}}, |event|'s {{FenceEvent/eventData}},
+and [=this=]'s [=relevant global object=]'s [=associated Document=].
+
+Modify <a spec=fenced-frame>report an event</a> as follows:
+
+Add a [=Document=] parameter called |document|.
+
+Replace the step
+
+> <a spec=fenced-frame>Send a beacon</a> with <var ignore>reporting map</var>[|destination|],
+> |eventType|, and |eventData|.
+
+With
+
+<a spec=fenced-frame>Send a beacon</a> with <var ignore>reporting map</var>[|destination|],
+|eventType|, |eventData|, and |document|.
+
+Modify [=finalize a reporting destination=] as follows:
+
+Add a [=Document=] parameter called |document|.
+
+Replace the step
+
+> <a spec=fenced-frame>Send a beacon</a> with |destination map|, |pending event|'s [=pending event/eventType=],
+> and |pending event|'s [=pending event/eventData=].
+
+With
+
+<a spec=fenced-frame>Send a beacon</a> with |destination map|, |pending event|'s [=pending event/eventType=],
+|pending event|'s [=pending event/eventData=], and |document|.
+
+Modify <a spec=fenced-frame>send a beacon</a> as follows:
+
+Add a [=Document=] parameter called |document|.
+
+After the step
+
+> Let <var ignore>destination url</var> be <var ignore>destination map</var>[|eventType|].
+
+add the steps
+
+1. Let |attributionReportingEligibility| be "<code>[=eligibility/unset=]</code>".
+1. If the "`attribution-reporting`" feature <a href="https://w3c.github.io/webappsec-permissions-policy/#algo-is-feature-enabled">is enabled</a>
+     in |document| with |document|'s [=origin=] and the result of [=getting supported registrars=]
+     is not [=set/is empty|empty=]:
+         1. Set |attributionReportingEligibility| to "<code>[=eligibility/event-source=]</code>".
+
+         Issue: Allow "<code>[=eligibility/navigation-source=]</code>" when automatic beacon is specified.
```

Comment:
Is there an issue on the Fenced Frame spec we can link to instead?

Reply:
Not yet, chatted with fenced-frame team earlier to add an issue for that.